spec/controllers/content_blocks_spec.rb

```diff
@@ -28,6 +38,54 @@ def check_object(blocks, index, content_block)
         check_object(blocks, 0, content_block)
         check_object(blocks, 1, content_block2)
       end
+
+      describe 'cache behaviour' do
+        context 'on the first request' do
+          it 'loads the content-blocks and returns a 200' do
+            get :index, api_key: content_block.api_key
+            expect(response).to have_http_status 200
+            etag_and_last_modified_available?
```

Comment:
does this method return a boolean? if not, i don't expect it to end with "?" ;)
=> suggestion: rename to expect_etag_and_last_modified_headers